ASP.NET Core/Web/AspNetCoreTemplate.Web/Areas/Identity/Pages/Account/Logout.cshtml.cs

```diff
@@ -0,0 +1,44 @@
+﻿namespace AspNetCoreTemplate.Web.Areas.Identity.Pages.Account
+{
+    using System.Threading.Tasks;
+
+    using AspNetCoreTemplate.Data.Models;
+    using Microsoft.AspNetCore.Authorization;
```

Comment:
Blank line before this namespace group

Reply:
https://github.com/NikolayIT/ASP.NET-MVC-Template/pull/36/commits/45af7c6192ffbd42fa7f157cd2469bb5293413b5